docs/documentation/getting-started/deployment/infrastructure-as-code/index.md

```diff
@@ -1,8 +1,8 @@
 # Infrastructure as Code
 
 !!! warning
-    Carbyne Stack Infrastructure as Code (IaC) is still in *proof-of-concept*
-    stage. Reach out in case you encounter problems.
+Carbyne Stack Infrastructure as Code (IaC) is still in *proof-of-concept*
```

Comment:
must be indented or the text is not rendered within the box
![image](https://github.com/carbynestack/carbynestack.github.io/assets/43000614/2b911967-28ef-4868-9407-bbbe0bcd96b5)